src/BooNEHadronInelasticProcess.cc

```diff
@@ -300,6 +300,17 @@ BooNEHadronInelasticProcess::PostStepDoIt(const G4Track& aTrack, const G4Step&)
       secTrack->SetUserInformation(tInfo);
     }
     tInfo->SetCreatorModelName(GetHadronicInteraction()->GetModelName()); 
+
+    // Propagate QE info for secondaries created in BooNEpBeInteraction to the track info
+    if (fInteraction->GetModelName() == "BooNEpBeInteraction") {
+      //std::cout << "    --> HadronInelasticProcess QE:" << fBooNEpBeModel->QuasiElasticStatus() << " for track E " << secTrack->GetKineticEnergy() << " PDG: " << secTrack->GetParticleDefinition()->GetPDGEncoding() << std::endl;
```

Comment:
Could you remove the debug comment please?

Reply:
forgot that, done!